doc/api.rst

```diff
@@ -222,7 +224,7 @@ Benchmark class
 
       Get the total number of values.
 
-   .. method:: get_nwarmup() -> int or float
```

Comment:
Please keep the information about the return type. Would it work to use `int | float`?

Same remark for other methods below.